source/print.c

```diff
@@ -0,0 +1,55 @@
+// Copyright 2024 10xEngineers
+
+#include "../common/print.h"
```

Comment:
Includes should not include full relative path, it should be possible to compile just using the header name, as long as the file has been included in the makefile.

Reply:
resolved this relative path issue. Now code is compiled just using the header name.